src/main/java/httpserver/HttpStatus.java

```diff
@@ -0,0 +1,9 @@
+package httpserver;
+
+public final class HttpStatus {
+    public static final String OK = "200 Ok";
+    public static final String METHODNOTALLOWED = "405 Method Not Allowed";
+    public static final String BADREQUEST = "400 BAD REQUEST";
```

Comment:
Not absolutely necessary but would be easier to read if multiple words were separated by an underscore here

Reply:
Much clearer with the underscores. Thanks.